lib/airbrake/capistrano/tasks.rb

```diff
@@ -6,7 +6,8 @@
   namespace :airbrake do
     desc "Notify Airbrake of the deploy"
     task :deploy do
-      on roles(:all) do
+      role = roles(:all, select: :primary).first || roles(:all).first
```

Comment:
Thanks for the explanation, that does make sense. That said, how confident are you in this? Why not to do:

```ruby
# If you want to opt in this new behaviour, you define your `:primary`,
# otherwise we just fall back to how it worked before.
roles(:all, select: :primary).first || roles(:all)
```

I'm just surprised that it wasn't an issue for all these years, so `roles(:all).first` _seems_ to be a safer solution.